CONTRIBUTING_FR.md

```diff
@@ -4,4 +4,137 @@
+
+ **Note : toutes les PR vers d'autres branches que `master` seront refusées.**
+
+Les administrateurs du projet effectueront une revue de votre code avant de fusionner votre travail. Selon le type de projet, des tests et autres opérations automatisées pourront être mises en place afin de s'assurer de la robustesse de votre production. Si un problème apparait, ou si nous remarquons un détail à modifier, vous pourrez ajouter des commits sur votre propre branche de travail afin d'effectuer les corrections nécessaires : elles seront automatiquement ajoutées à votre `pull request`.
+
+## Contribution régulière au code source
+
+**Note :** cette procédure s'adresse aux membres du GSRI ou aux contributeurs réguliers identifiés. Ils·elles sont autorisés à contribuer directement dans notre dépôt afin d'éviter la prolifération des forks. Si vous n'êtes pas autorisés à contribuer directement au dépôt, référez-vous aux instructions de la section *Contribution ponctuelle au code source* plus haut dans ce document.
+
+### Récupérer le droit d'écriture sur le dépôt
+
+Votre première étape pour les contributions directes est d'obtenir les droits d'accès sur notre dépôt. Prenez contact avec nous sur Discord et nous étudierons votre demande.
+
+### Cloner le dépôt
+
+Si vous êtes un contributeur régulier, vous pouvez contribuer directement au dépôt. Vous n'avez pas besoin de créer de fork de notre dépôt : vous pouvez cloner directement le dépôt qui vous intéresse.
+
+### Créer une nouvelle branche
+
+**Note : cette étape est obligatoire.** Ne pas s'y circonvenir vous permettra certes de créer des commits sur master **localement**, mais toute tentative de `push` vers Github sera automatiquement refusée.
+
+Afin de créer une branche, sélectionnez `Branch` / `New branch ...` dans le menu. Github Desktop demandera un nom de branche. Entrez le nom souhaité : pseudo, fonctionnalité... Dans le doute, demandez-nous si le nom choisi est pertinent. Assurez-vous que la branche source est `master`. Une fois créée, publiez la branche sur Github en cliquant sur le bouton dans la barre en haut de Github Desktop.
+
+Les autres opérations sont similaires à celles décrites dans la procédure ci-dessus.
+
+### Garder votre branche à jour avec master
+
+Il est recommandé de vérifier régulièrement que vous ne travaillez pas sur une version obsolète des sources. Vérifiez que votre branche n'a aucun commit *behind* (en retard) relativement à notre branche `master`. Si votre branche n'est pas à jour avec `master`, l'ouverture d'une `pull request` sera refusée par notre équipe. Si des changements ont eu lieu sur la branche `master`, vous pouvez utiliser la fonction `Branch` / `Update from master` de Github Desktop afin de rattraper le retard. La résolution des conflits doit être faite sur votre actuelle branche de travail.
```

Comment:
```suggestion
Il est recommandé de vérifier régulièrement que vous ne travaillez pas sur une version obsolète des sources. Vérifiez que votre branche n'a aucun commit *behind* (en retard) relativement à notre branche `master`. Si votre branche n'est pas à jour avec `master`, l'ouverture d'une `pull request` sera refusée par notre équipe. Si des changements ont eu lieu sur la branche `master`, vous pouvez utiliser la fonction `Branch` / `Update from master` de Github Desktop afin de rattraper le retard. La résolution des conflits doit être faite sur votre branche de travail.
```

Reply:
If I'm right, there is no change in this suggestion ?